src/index.js

```diff
@@ -52,7 +53,8 @@ const launchBrowserIfNeeded = async function ({ getBrowser }) {
         // for spammy pages that keep on sending (non critcal) requests
         waitUntil: 'networkidle2',
         ignoreHTTPSErrors: true,
-        args: ['--disable-setuid-sandbox', '--no-sandbox']
+        args: ['--disable-setuid-sandbox', '--no-sandbox'],
+        dumpio: false
```

Comment:
This seems to be the default in the [latest version of Puppeteer](https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#puppeteerlaunchoptions). Why did you add it; is it not the default on the 0.12 version?

Reply:
I missed to remove this. Just used it to dump all console output while searching for the execution context problem. Can be removed